src/share/poudriere/image.sh

```diff
@@ -58,7 +58,8 @@ Options:
                        the disk image and mount it to \$WRKDIR/world before
                        installing the contents to the image
     -c overlaydir   -- The content of the overlay directory will be copied into
-                       the image
+                       the image. Owners and permissions will be overwritten if
+                       an overlaydir.mtree file is found
```

Comment:
A bit confusing as people might think that the file needs to be strictly named 'overlaydir.mtree'

Reply:
Could using `$overlaydir.mtree` enough to solve this confusion ?

Reply:
Or <overlaydir>.mtree

Reply:
Stupid github, that was supposed to be \<overlaydir\>.mtree